mitype/keycheck.py

```diff
@@ -21,6 +22,13 @@ def is_escape(key):
     return False
 
 
+def is_ignored(key):
+    if sys.version_info[0] < 3:
+        return key[:3] == "KEY"
```

Comment:
On linux, there are some other invalid keys such as kpADD.
So to avoid this, lets change this line to -
`return key.startswith("KEY") or (len(key) > 1 and key.startswith("k"))`

Reply:
Hey @Mithil467 

I've made those changes, please review them.